imports/plugins/included/inventory/server/methods/statusChanges.js

```diff
@@ -64,6 +64,16 @@ Meteor.methods({
 
     // update inventory status for cartItems
     for (const item of cartItems) {
+      if (reservationStatus === "sold") {
+        const _id = item.productId;
+        const product = Products.findOne(_id);
+        if (product) { // check that a product with the _id exists so we don't end up creating a new product with upsert
```

Comment:
### Nit-Pick
I think it is better to write your comment before the "IF" condition